src/components/TextInput/BaseTextInput.js

```diff
@@ -255,17 +255,19 @@ class BaseTextInput extends Component {
                                     pointerEvents="box-none"
                                 >
                                     {Boolean(this.props.prefixCharacter) && (
-                                        <Text
-                                            pointerEvents="none"
-                                            selectable={false}
-                                            style={[
-                                                styles.textInputPrefix,
-                                                !hasLabel && styles.pv0,
-                                            ]}
-                                            onLayout={this.storePrefixLayoutDimensions}
-                                        >
-                                            {this.props.prefixCharacter}
-                                        </Text>
+                                        <View style={styles.textInputPrefixWrapper}>
```

Comment:
Why do we need this View?

Reply:
To make it align center for native we need this parent `View` as `Text` won't render an extra tag(like `div` gets created for the web). Also, I think it is good to apply height and other properties to the `View` which isn't related to the `Text` component.